### PR TITLE
Issue #1323

### PR DESCRIPTION
**Pull request by j0rt3g4**
Updated Links and text #1323

### CODE REVIEW

**Comment by scanum** [comment]
@nenonix @JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: OneDrive/sync-process.md [hunk 1]
```diff
@@ -46,6 +46,14 @@ Here’s how it works:
 
 The previous OneDrive for Business sync app (Groove.exe) used a polling service to check for changes on a predetermined schedule. Polling can lead to system lag and slowness because it requires a lot of computing power. Using WNS is a significant enhancement.
   
+
+## Auhentication protocols
+
+The authentications protocols depend Sharepoint version you are using
```
Comment (JohanFreelancer9) [inline OneDrive/sync-process.md]:
```suggestion
The authentication protocols depend on which version of SharePoint you are using.
```

File: OneDrive/sync-process.md [hunk 2]
```diff
@@ -46,6 +46,14 @@ Here’s how it works:
...
+
+ - For Sharepoint Server 2019, uses NTLM.
```
Comment (JohanFreelancer9) [inline OneDrive/sync-process.md]:
```suggestion
 - Sharepoint Server 2019 uses NTLM.
```

File: OneDrive/sync-process.md [hunk 3]
```diff
@@ -46,6 +46,14 @@ Here’s how it works:
...
+ - For Sharepoint Online, uses fed auth.
```
Comment (JohanFreelancer9) [inline OneDrive/sync-process.md]:
```suggestion
 - SharePoint Online uses FedAuth.
```

File: OneDrive/sync-process.md [hunk 4]
```diff
@@ -67,3 +75,8 @@ The OneDrive sync app uses the following to sync files:
 ## Security and encryption
 
 File chunks are stored in multiple containers in Azure, each of which is given a unique key. Each key is required to reassemble the complete file. There’s also a separate master key encrypting each file chunk key, ensuring the data remain secure even when not moving.
+
+## Related Seccion
```
Comment (JohanFreelancer9) [inline OneDrive/sync-process.md]:
```suggestion
## Related topic
```

File: OneDrive/sync-process.md [hunk 5]
```diff
@@ -67,3 +75,8 @@ The OneDrive sync app uses the following to sync files:
...
+
+- [Sharepoint Authentication](https://docs.microsoft.com/sharepoint/authentication)
```
Comment (JohanFreelancer9) [inline OneDrive/sync-process.md]:
```suggestion
- [SharePoint Online Authentication](https://docs.microsoft.com/sharepoint/authentication)
```

**Pull request merged**